README.md

```diff
@@ -1,70 +1,196 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Walkthrough to the Leaderboard Scores](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# 📖 [Stock Quote] <a name="about-project"></a>
+This App is developed to show real time stock quotes base on the price, change percentages and other  parameters. It is design to determine how the market trends moves.
+**[Stock Quote]** 
+
+## 🛠 Built With <a name="built-with"></a>
 
-## Available Scripts
+### Tech Stack <a name="tech-stack"></a>
 
-In the project directory, you can run:
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3.org/standards/webdesign/htmlcss.html#whathtml">HTML</a></li>
+    <li><a href="https://www.w3.org/standards/webdesign/htmlcss.html#whatcss">CSS</a></li>
+    <li><a href="https://www.w3.org/standards/webdesign/script">JavaScript</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://jestjs.io/">Jest</a></li>
+    <li><a href="https://react.dev/">React</a></li>
+    <li><a href="https://react-redux.js.org/">Redux</a></li>
+  </ul>
+</details>
+  
+### Key Features <a name="key-features"></a>
 
-### `npm start`
+**Enlist skill sets and tech stack**
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- Use React documentation.
+- Use React components.
+- Use React props.
+- Use React Router.
+- Connect React and Redux whit API.
+- Handle events in a React app.
+- Write unit tests with React Testing Library.
+- Use styles in a React app.
+- Use React hooks.
+- Apply React best practices and language style guides in code.
+- Use store, actions and reducers in React.
+- Use the [Stock Quote](https://site.financialmodelingprep.com/developer/docs/stock-api/) API
+- Perform a code review for a team member.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### `npm test`
+## 🚀 Live Demo <a name="live-demo"></a>
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+ <a href="https://grr-lhtg.onrender.com">Click here</a> to view live version 
+  <!--  This feature is not implemented yet  -->
```

Comment:
- [ ] Just as the previous reviewer highlighted, kindly show some piece of code that you're proud of implementing. It's required to show a piece of code you're proud of. Please redo the video and take note of all the points highlighted by the previous reviewer 👍🏽